include/fota_download.rst

```diff
@@ -0,0 +1,19 @@
+.. _lib_fota_download:
+
+Firmware Over the Air Download
+##############################
+
+The Firmware Over the Air Download library provides functions to download a firmware file as an update candidate to MCUboots secondary slot.
```

Comment:
Mention HTTP in the docs.

Reply:
The download client should be transport agnostic, so IMO we should keep it like this to avoid having to provide future updates.

I know HTTPS is planned.